server/models/sentReferralDashboard.ts

```diff
@@ -0,0 +1,16 @@
+import User from './hmppsAuth/user'
+import ServiceProvider from './serviceProvider'
+import ServiceUser from './serviceUser'
+
+export default interface SentReferralDashboard {
```

Comment:
as we discussed in the backend PR, we are going to rename this class.

Reply:
I have renamed this class and all the reference to this class too